thoth/common/openshift.py

```diff
@@ -1430,6 +1430,34 @@ def schedule_kebechet_workflow(
             },
         )
 
+    def schedule_kebechet_administrator_workflow(
```

Comment:
didn't we want to remove `_workflow` in front in the method name? we have mixed naming, in theory method in Openshift class should hide the logic, because if we don't use Argo in the future we need to rechange name to something else rather than a workflow :)

Reply:
Thanks I have changed it!